Automattic/jetpack
# Add/golden token experience

Adds a Golden Token experience to My Jetpack. 

Alternatively, here's a version that only loads in Jetpack and can be done on any page in wp-admin: https://github.com/Automattic/jetpack/compare/add/golden-token-modal-basic?expand=1 

## Proposed changes:

- New route in My Jetpack: `#/redeem-token`
- Shows a cool animation if the site has a partner token. 

### Other information:
- [Fix] Updates the `isRequestingPurchases` selector and test in My Jetpack to actually return state of fetching purchases.
- [Added] New root vars in jetpack-base-style package. 
- This PR does _not_ introduce any UI or direct links to the new route. That will come later. 
- The SCSS is a bit... fragile. 

## Jetpack product discussion



p1HpG7-ljt-p2

## Does this pull request change what data or activity we track or use?


Some basic tracks. 

## Testing instructions:






- Connect the site to your wpcom user account
- Use the MC tool jetpack/golden-token to give the site a "token"
- go to `/wp-admin/admin.php?page=my-jetpack#/redeem-token`
- Check out the fun.
